sim/hunter/items.go

```diff
@@ -569,6 +570,124 @@ func init() {
 		hunter.ItemSwap.RegisterProc(SignetOfTheUnseenPath, procAura)
 	})
 
+	core.NewItemEffect(StringsOfFate, func(a core.Agent) {
+		hunter := a.(HunterAgent).GetHunter()
+
+		hasMeleeSpecialist := hunter.HasRune(proto.HunterRune_RuneBeltMeleeSpecialist)
+
+		// After using the bow active you cannot gain strands for 30sec.
+		noGainAura := hunter.GetOrRegisterAura(core.Aura{
+			Label:    "Strand Of Fate - No Stack Gain",
+			ActionID: core.ActionID{SpellID: 1231603},
+			Duration: time.Second * 30,
+		})
```

Comment:
We should just be able to build an ICD for this without a dedicated aura. I could see the benefit of the aura if it would be clear to users when it's on GCD but the wowhead tooltip isn't very useful
```suggestion
		icd := core.Cooldown{
		  Timer: hunter.NewTimer()
		  Duration: time.Second * 30
		}
```